react/src/demo/example-data/deckgl-map.json

```diff
@@ -73,6 +74,21 @@
         "rotDeg": 0,
         "image": "@@#resources.depthMap"
       },
+      {
+        "@@type": "GridLayer",
+        "id": "grid-layer",
+        "data": "https://raw.githubusercontent.com/equinor/webviz-subsurface-components/master/react/src/demo/example-data/grid_layer.json",
+        "colorMapName": "Rainbow",
+        "valueRange": [
+          0,
+          1
+        ],
+        "colorMapRange": [
+          0,
+          1
+        ],
+        "visible": true
```

Comment:
This leaves the grid layer hidden behind the property map layer in the default storybook example (with a small artifact protruding on the left side of the map). Would be better to create a separate storybook example for the grid layer.